core/dagop.go

```diff
@@ -718,28 +689,3 @@ func newDagOpLLB(ctx context.Context, dagOp buildkit.CustomOp, id *call.ID, inpu
 		buildkit.WithPassthrough(),
 	)
 }
-
-// loadDagOpID loads the target ID from the server.
-//
-// It takes two contexts, the first to load the receiver with, the second to
-// load the current call with. This *ensures* that we aren't accidentally
-// applying the dagop to multiple calls - only the most recent.
-func loadDagOpID(ctx context.Context, loadCtx context.Context, srv *dagql.Server, id *call.ID) (dagql.Typed, error) {
```

Comment:
@jedevc this was a surprise nice side effect; we can get rid of the little "dance of two contexts" here since now whether something should run in dag-op context is determined by args in the call rather than the actual `context.Context`